tools/cli/go-whisk/whisk/client.go

```diff
@@ -254,6 +260,12 @@ func bodyTruncator(body io.ReadCloser) (string, io.ReadCloser, error) {
 func (c *Client) Do(req *http.Request, v interface{}, ExitWithErrorOnTimeout bool) (*http.Response, error) {
     var err error
     var truncatedBody string
+    obfuscateArr := []ObfuscateSet{
```

Comment:
@jessealva Let's also enable this array to be configured by the user of this http client module.  that is, if the caller knows that it expects a one or more sensitive strings to be part of either the request or response, then the caller will add these sensitive strings to this array so that they are replace anywhere in the output.